st2web/Dockerfile

```diff
@@ -76,7 +76,8 @@ RUN if [ "${ST2_VERSION#*dev}" != "${ST2_VERSION}" ]; then \
 # It's a user's responsbility to pass the valid SSL certificate files: 'st2.key' and 'st2.crt', used in nginx
 VOLUME ["/etc/ssl/st2/"]
 
-EXPOSE 80
-EXPOSE 443
+EXPOSE 8080
+EXPOSE 8443
```

Comment:
Does docker allow referencing an `ARG` here? If so, then people could choose to use root anyway and adjust this accordingly.

Reply:
Unsure - but AFAIK (I could very well be wrong) - that would make it only configurable on build of the container, rather than deploy - which would be annoying.

I've just gone ahead and exposed both `80`/`443` _and_ `8080`/`8443` so that backwards compatibility is maintained while allowing the use of non-privileged ports

Reply:
Hmm. I guess EXPOSE is just for documentation so exposing both is fine: https://docs.docker.com/reference/dockerfile/#expose